test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -135,7 +135,7 @@ def docker_get(args, container_id, src, dst):
                     options=['-i'], stdout=dst_fd, capture=True)
 
 
-def docker_run(args, image, options, cmd=None):
+def docker_run(args, image, options, cmd=None, create_only=False):
```

Comment:
Add `create_only` to the type hints in the docstring.

Reply:
done.